### PR TITLE
[openshift-resources] Fix for html_url in Prometheus alerts

### PR DESCRIPTION
**Pull request by cubismod**
A second attempt at #3990 

Some colleagues pointed out that the earlier attempt resulted in a doubling of qontract-server requests which means more $$$. This take removes that additional GQL call as the function already has a settings dict passed to it which includes the `repoUrl` value I was previously querying for in GQL.

### CODE REVIEW

File: reconcile/openshift_resources_base.py [hunk 1]
```diff
@@ -44,7 +67,10 @@
 from reconcile.utils.secret_reader import SecretReader
 from reconcile.utils.semver_helper import make_semver
 from reconcile.utils.sharding import is_in_shard
-from reconcile.utils.vault import SecretVersionIsNone, SecretVersionNotFound
+from reconcile.utils.vault import (
+    SecretVersionIsNone,
+    SecretVersionNotFound,
+)
```
Comment (rporres) [inline reconcile/openshift_resources_base.py:73]: why this lint changes? where do they come from?

Comment (cubismod) [reply reconcile/openshift_resources_base.py:73]: I think in my previous PR I had ruff setup incorrectly with the vscode extension. When I ran `make format` this time, it outputted these changes

**Review by rporres — COMMENTED** [review]
Looks good except for those linting changes that I don't know where they come from...

File: reconcile/openshift_resources_base.py (hunk 1 above)
Comment (rporres) [reply reconcile/openshift_resources_base.py:73]: weird, I run `make format` locally and I don't get any change...

Comment (cubismod) [reply reconcile/openshift_resources_base.py:73]: User error on my part :rofl:

**Review by rporres — APPROVED** [review]
anyway, it's fine... The current import syntax looks the most used in the code.

File: reconcile/openshift_resources_base.py (hunk 1 above)
Comment (rporres) [reply reconcile/openshift_resources_base.py:73]: it's fine, it got me a bit confused. But the new syntax is the most widely used in the code.

**Review by patjlm — APPROVED** [review]

**Pull request merged**